app/views/planning_applications/review/tasks/_review_conditions.html.erb

```diff
@@ -1,28 +1,56 @@
 <%= bops_task_accordion(id: "review-conditions-section") do |accordion| %>
   <% accordion.with_heading(text: "Review conditions") %>
+  <% if @planning_application.review_heads_of_terms? %>
+    <% accordion.with_section(id: "review-heads-of-terms", expanded: false) do |section| %>
```

Comment:
Should this be `expanded: false` ?

Reply:
Think we want to add this conditionally if an error is present or at least not hardcode this value